R/style.R

```diff
@@ -23,13 +24,13 @@ readLegend <- function(legend_file = NULL){
   )
 
   dup <- which(duplicated(csv$Short))
-  if(length(dup) > 0){
-    warning(paste0("There are duplicated values: ", paste0(csv$Short[dup], collapse = ", ")))
+  if (length(dup) > 0) {
+    stop(paste0("There are duplicated short names: ", paste0(csv$Short[dup], collapse = ", ")))
   }
 
-  dup <- which(duplicated(csv$Color))
```

Comment:
It is interesting to show duplicated values for colors as well.

Reply:
I've changed this because a parent label could have same color of one of child label. For example, Natural->Forest Natural->Forest Amazon Biome could share same color. However, I agree that a warning can avoid non-intentional same color among labels.